dashboard/app/models/census2017v3.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: census_submissions
+#
+#  id                           :integer          not null, primary key
+#  type                         :string(255)      not null
+#  submitter_email_address      :string(255)
+#  submitter_name               :string(255)
+#  submitter_role               :string(255)
+#  school_year                  :integer          not null
+#  how_many_do_hoc              :string(255)
+#  how_many_after_school        :string(255)
+#  how_many_10_hours            :string(255)
+#  how_many_20_hours            :string(255)
+#  other_classes_under_20_hours :boolean
+#  topic_blocks                 :boolean
+#  topic_text                   :boolean
+#  topic_robots                 :boolean
+#  topic_internet               :boolean
+#  topic_security               :boolean
+#  topic_data                   :boolean
+#  topic_web_design             :boolean
+#  topic_game_design            :boolean
+#  topic_other                  :boolean
+#  topic_other_description      :string(255)
+#  topic_do_not_know            :boolean
+#  class_frequency              :string(255)
+#  tell_us_more                 :string(255)
+#  pledged                      :boolean
+#  created_at                   :datetime         not null
+#  updated_at                   :datetime         not null
+#
+# Indexes
+#
+#  index_census_submissions_on_school_year_and_id  (school_year,id)
+#
+
+class Census2017v3 < Census2017v2
+end
```

Comment:
We have 3 versions already?

Can you add some description here so it's clear what these different versions mean?

Reply:
There are already two major versions (/yourschool and HoC signup) each with multiple variations. I'll add comments to explain the different versions.

Reply:
Census data exists in multiple versions in the FORMS table:
* Census2017 (from /yourschool) 
  * Varitaion 1: without NCES school ids, no “other description”
  * Variation 2: no NCES school id, with “other description”
  * Variation 3: with NCES school id, with “other description”
* Hoc signup 
  * HocSignup2017 (prior to census questions being separated from HOC signup)
  * HocCensus2017 
    * Variation 1: without NCES school ids
    * Variation 2: with NCES school ids

Reply:
So, do we have a Ho 2017 version 3 yet? I'd rather not create empty classes, unless they're actually in use. It's easy to create them in the future when needed.